src/components/DataGraph/util.js

```diff
@@ -35,13 +40,92 @@ var parseC3Data = function(data) {
                     }
                     yAxisCount++;
                 }
-            }              
+            }          
         }
         allModelsData['columns'].push(xSeries);
         allModelsData['columns'].push(dataSeries);          
     }
-
     return [allModelsData, axisInfo];
 }
 
-export default parseC3Data
\ No newline at end of file
+
+var parseTimeSeriesForC3 = function(graph_data) {
+
+    var modelName = String(graph_data['id']);
+    var yUnits = String(graph_data['units']);
+    var C3Data = {
+        columns:[], 
+        types: {
+            modelName: 'line', 
+            'Annual Average': 'step',
+            'Seasonal Average': 'step'
+        }, 
+        labels: {
+            format: {
+                'Seasonal Average': function (v, id, i, j){
+                    if (i == 0 || i == 11){ return "Winter" }
+                    if (i == 3) { return "Spring" }
+                    if (i == 6) { return "Summer" }
+                    if (i == 9) { return "Fall" }
+                }
+            }
+        },
+        axes: {modelName:'y'},
+    };
+
+    var axisInfo = { 
+        x: { type:'category', categories:[] },
+        y: { label: { 'text': yUnits, 'position':'outer-middle' }} 
+    };
+
+    var tooltipInfo = {
+        grouped: true,
+        format: {
+            value: function (value) { return value + ' ' + yUnits }
+        } 
+    };
+
+    var monthlySeries = [modelName];
+    var springSeries = [];
+    var summerSeries = [];
+    var fallSeries = [];
+    var winterSeries = [];
+    var seasonalLabel = ['Seasonal Average'];
+    var annualSeries = ['Annual Average'];
+
+    var idx = 0;
+    for (let key in graph_data['data']) {
+        var val = graph_data['data'][key].toFixed(PRECISION);
+        var timestep = moment(key, moment.ISO_8601);
+        var month = timestep.format('MMMM');
+        if (idx < 12){
+            axisInfo['x']['categories'].push(month);
+            monthlySeries.push(val);
+        }
+        else if (idx === 12){
+            winterSeries.push(val, val, val);
+        }
+        else if (idx === 13){
+            springSeries.push(val, val, val);
+        }
+        else if (idx === 14){
+            summerSeries.push(val, val, val);
+        }
+        else if (idx === 15){
+            fallSeries.push(val, val, val);
+        }
+        else if (idx === 16){
+            annualSeries.push(val, val, val, val, val, val, val, val, val, val, val, val);
```

Comment:
This is pretty bulky... with underscore you can reduce this with something like `_.times(5, function(){return this}, val)`